src/components/graphs/PieChart.vue

```diff
@@ -35,6 +44,14 @@ export default class PieChart extends Vue {
   readonly graphMargins = { top: 0, right: 0, bottom: 0, left: 0 };
 
   svg!: d3.Selection<SVGGElement, unknown, HTMLElement, any>;
+  tooltipMessage = `
+    <h6>Why does this matter?</h6>
```

Comment:
A few things:

1. The pie chart is meant to be generic (see the doc block) and could be used for anything. I'd recommend putting the tooltip outside the pie chart, and on the page that uses it instead.

2. I don't think you want to use an `<h6>` here - if this is a discrete focused `<dialog>` element or something like that, it should be an `<h1>` (because it's the top if it's element). Otherwise I'd just use CSS, like a `p class="title"` to style this and mirror the `h6` styling, since you can't know for sure that the heading above this one is an `h5`, and headings have to be in order on a page. See: https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks